packages/serverpod/lib/src/relic/web_server.dart

```diff
@@ -4,12 +4,22 @@ import 'dart:io';
 
 import 'package:path/path.dart' as path;
 import 'package:serverpod/serverpod.dart';
+import 'package:serverpod/src/relic/static/static_handler.dart';
 
 /// The Serverpod webserver.
 class WebServer {
   /// Reference to the [Serverpod] this webserver is associated with.
   final Serverpod serverpod;
 
+  /// If a security context is provided an HTTPS server will be started.
+  final SecurityContext? securityContext;
+
+  /// The [address] can either be a [String] or an [InternetAddress]. If
+  /// [address] is a [String], the server will perform a
+  /// [InternetAddress.lookup] and use the first value in the list. Defaults to
+  /// [InternetAddress.anyIPv6].
+  final Object? address;
```

Comment:
I don't really like the unspecified type, but it may be ok as the underlaying `HttpServer` uses `dynamic`.

Reply:
+1 Would also prefer that this was typed.